ols/app/metrics/metrics.py

```diff
@@ -11,9 +11,10 @@
     generate_latest,
 )
 
-from ols.utils.auth_dependency import auth_dependency
+from ols.utils.auth_dependency import AuthDependency
 
 router = APIRouter(tags=["metrics"])
+auth_dependency = AuthDependency(virtual_path="/ols-access")
```

Comment:
should be /ols-metrics-access

Reply:
indeed looks like i mixed that durin cut/paste with the feedback one